wagtail_ab_testing/wagtail_hooks.py

```diff
@@ -70,28 +85,54 @@ def register_create_abtest_action_menu_item():
 
 # This is the only way to inject custom JS into the editor with knowledge of the page being edited
 class AbTestingTabActionMenuItem(ActionMenuItem):
-    def render_html(self, request, context):
-        if 'page' in context:
-            return format_html(
-                '<script src="{}"></script><script src="{}"></script><script>window.abTestingTabProps = JSON.parse("{}");</script>',
-                reverse('wagtail_ab_testing_admin:javascript_catalog'),
-                versioned_static('wagtail_ab_testing/js/wagtail-ab-testing.js'),
-                escapejs(json.dumps({
-                    'tests': [
-                        {
-                            'id': ab_test.id,
-                            'name': ab_test.name,
-                            'started_at': ab_test.first_started_at.strftime(DATE_FORMAT) if ab_test.first_started_at else _("Not started"),
-                            'status': ab_test.get_status_description(),
-                            'results_url': reverse('wagtail_ab_testing_admin:results', args=[ab_test.page_id, ab_test.id]),
-                        }
-                        for ab_test in AbTest.objects.filter(page=context['page']).order_by('-id')
-                    ],
-                    'can_create_abtest': request.user.has_perm('wagtail_ab_testing.add_abtest'),
-                }))
-            )
-
-        return ''
+    if WAGTAIL_VERSION >= (2, 15):
+        # see if this is a pre-2.15 
+        def render_html(self, context):
+            if 'page' in context:
+                return format_html(
```

Comment:
suggestion: move this to own method that accepts the user and context as params, then the two `render_html` definitions will be nice short, sweet and readable